docs/content/development/rest/_index.md

```diff
@@ -10,23 +10,21 @@ card:
 
 You need two HTTP Headers to request CDS API:
 
-- `Session-Token: your-cds-token`
+- `Authorization: Bearer your-cds-token`
 - `X-Requested-With: X-CDS-SDK`
 
 ```bash
 # List CDS Project
-curl -H "Session-Token: your-cds-token" -H "X-Requested-With: X-CDS-SDK" https://your-cds-api/project
+curl -H "Authorization: Bearer your-cds-token" -H "X-Requested-With: X-CDS-SDK" https://your-cds-api/project
 ```
 
 ## About CDS Token
 
-The CDS UI uses a non-persistent token. If you want to play with CDS API, you probably need a persistent token.
+The CDS UI uses a session token. If you want to play with CDS API, you probably need a persistent token.
```

Comment:
We should not say "The CDS UI uses a session token. If you want to play with CDS API, you probably need a persistent token".
Because now all CDS API consumer are using consumers